master/pkg/searcher/adaptive_asha_test.go

```diff
@@ -25,194 +27,65 @@ func TestBracketMaxConcurrentTrials(t *testing.T) {
 	assert.DeepEqual(t, getBracketMaxConcurrentTrials(0, 4., []int{40, 10}), []int{10, 10})
 }
 
-func modePtr(x expconf.AdaptiveMode) *expconf.AdaptiveMode {
-	return &x
-}
-
-func TestAdaptiveASHASearcherReproducibility(t *testing.T) {
-	conf := expconf.AdaptiveASHAConfig{
-		RawMaxLength: ptrs.Ptr(expconf.NewLengthInBatches(6400)),
-		RawMaxTrials: ptrs.Ptr(128),
-	}
-	conf = schemas.WithDefaults(conf)
-	gen := func() SearchMethod { return newAdaptiveASHASearch(conf, true) }
-	checkReproducibility(t, gen, nil, defaultMetric)
-}
-
-func TestAdaptiveASHASearchMethod(t *testing.T) {
-	testCases := []valueSimulationTestCase{
+func TestMakeBrackets(t *testing.T) {
+	cases := []struct {
+		conf        expconf.AdaptiveASHAConfig
+		expBrackets []bracket
+	}{
 		{
-			name: "smaller is better",
-			expectedTrials: []predefinedTrial{
-				newConstantPredefinedTrial(toOps("300B 900B"), 0.1),
-				newConstantPredefinedTrial(toOps("300B"), 0.2),
-				newConstantPredefinedTrial(toOps("300B"), 0.3),
-				newConstantPredefinedTrial(toOps("900B"), 0.4),
-				newConstantPredefinedTrial(toOps("900B"), 0.5),
-			},
-			config: expconf.SearcherConfig{
-				RawSmallerIsBetter: ptrs.Ptr(true),
-				RawAdaptiveASHAConfig: &expconf.AdaptiveASHAConfig{
-					RawMaxLength: ptrs.Ptr(expconf.NewLengthInBatches(900)),
-					RawMaxTrials: ptrs.Ptr(5),
-					RawMode:      modePtr(expconf.StandardMode),
-					RawMaxRungs:  ptrs.Ptr(2),
-					RawDivisor:   ptrs.Ptr[float64](3),
+			conf: expconf.AdaptiveASHAConfig{
+				RawMode:                expconf.AdaptiveModePtr(expconf.StandardMode),
```

Comment:
Oh, this function is really old.  I wrote it before I wrote the generics-based `ptrs.Ptr()` (because we didn't have go generics at the time).

Please find all places where this function is used, and replace them with `ptrs.Ptr()`, please.